src/hotspot/cpu/s390/c2_MacroAssembler_s390.cpp

```diff
@@ -178,12 +179,12 @@ unsigned int C2_MacroAssembler::string_compress(Register result, Register src, R
     const int  log_min_vcnt = exact_log2(min_vcnt);
     Label      VectorLoop, VectorDone, VectorBreak;
 
-    VectorRegister Vtmp1      = Z_V16;
-    VectorRegister Vtmp2      = Z_V17;
-    VectorRegister Vmask      = Z_V18;
-    VectorRegister Vzero      = Z_V19;
-    VectorRegister Vsrc_first = Z_V20;
-    VectorRegister Vsrc_last  = Z_V23;
+    VectorRegister Vtmp1      = v16;
+    VectorRegister Vtmp2      = v17;
+    VectorRegister Vmask      = v18;
+    VectorRegister Vzero      = v19;
+    VectorRegister Vsrc_first = v20;
+    VectorRegister Vsrc_last  = v23;
```

Comment:
It would make more sense for the arguments to this function have names `Vtmp1`, `Vtmp2`, and so on. Rather than these aliases.

Reply:
Done.